gocourse02/main.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+)
+
+/*
+Написати програму “Зоопарк”. Звіри повтікали (більше трьох штук), наглядач повинен їх зібрати.
+Кожна сутність (наглядач, звір, клітка, …) представляється окремою структурою (zookeeper, animal, cage, …).
+Треба використати щонайменше: структури, вказівник, nil, будування, конструктор. Додати тваринам можливість
+розмножуватись. Програма має демонструвати свою роботу через вивід в stdout.
+*/
+
+const animalNumber = 15 // Кількість звірів які повтікали
+
+type ZooKeeper struct {
+	Name           string
+	NumberOfFound  int
+	NumberOfCaught int
```

Comment:
Може так краще? Трохи не зрозуміло, що таке `NumberOfFound`:
```suggestion
	AnimalsFound  int
	AnimalsCaught int
```